foodsaving/groups/templates/group_summary.mjml

```diff
@@ -59,13 +59,9 @@
                                     <em>{{ feedback.comment }}</em>
                                     <br>
                                     <span style="font-size: 14px;">
-                                        {% trans weight=feedback.weight, store_url=store_url(feedback.about.store), store_name=feedback.about.store.name %}
-                                            {{ weight }}kg
-                                            from
-                                            <a href="{{ store_url }}">
-                                                {{ store_name }}
-                                            </a>
-                                        {% endtrans %}
+                                    {% trans weight=feedback.weight, store_url=store_url(feedback.about.store), store_name=feedback.about.store.name -%}
```

Comment:
Found a trick to have nicer translation messages: if you write `{% trans -%}` (with a dash at the end), then Jinja2 trims whitespace.